packages/js/e2e-core-tests/src/specs/merchant/wp-admin-product-import-csv.test.js

```diff
@@ -17,11 +17,11 @@ const { it, describe, beforeAll, afterAll } = require( '@jest/globals' );
 const path = require( 'path' );
 const coreTestsPath = getCoreTestsRoot();
 const filePath = path.resolve(
-	coreTestsPath.packageRoot,
+	coreTestsPath.coreTestsRoot,
 	'test-data/sample_products.csv'
 );
 const filePathOverride = path.resolve(
-	coreTestsPath.packageRoot,
+	coreTestsPath.coreTestsRoot,
```

Comment:
We should be able to replace this with [`resolvePackagePath`](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/e2e-environment/utils/test-config.js#L79) and eliminate the `coreTestsRoot` property.

Reply:
Would it make sense to move `resolvePackagePath` to `e2e-utils`, to avoid `e2e-core-tests` depending on `e2e-environment`?

Reply:
@Luc45 Not in this PR but is something we may want to consider.

Reply:
Should I do anything in this PR related to `resolvePackagePath`? Should I duplicate it's logic to `e2e-core-tests` package, and create a follow-up issue to centralize this in `e2e-utils`?

Reply:
> Should I do anything in this PR related to resolvePackagePath?

I'll create a follow up issue as I didn't get a chance to look into the `appRoot`.

> Should I duplicate it's logic to e2e-core-tests package, and create a follow-up issue to centralize this function in e2e-utils

We would only do something like this if we had published packages with the function in one package then later decided to move it to another.